### DISCUSSION TITLE
JLReq TF Meeting Notes 2024-6-11

### DISCUSSION THREAD

**Discussion by kidayasuo** [w3c/jlreq#435] [meetingnotes]
# JLReq TF Meeting Notes 2024-6-11
（[日本語](https://github.com/w3c/jlreq/discussions/435#discussioncomment-9775218)）

## Review of Chapters 2 and 3 of jlreq-d
These chapters provide guidelines for writers. Unlike in print workflows, writers can directly control major aspects of text layout in digital formats.

#### General
Important aspects to cover for writers (bin-sensei):
- Usage of punctuation marks
- Vertical and horizontal writing
- Paragraph issues
- Headings and notes: to be written later if necessary
- Usage of Latin letters and Arabic numerals
All these are currently included in Chapter 2 (Kida).

Combine Chapters 2 and 3. It’s fine as long as we understand the separation of text and its formatting in our minds (Kobayashi-san).

#### Section 3.1 Paragraph Styles
- Block paragraphs: Change "Insert a blank line between paragraphs" to "Increase the spacing between paragraphs".
- Hanging indents: also used in notes and bullet points.
- It's challenging to clearly distinguish between alignment styles and paragraph styles. How should we differentiate them?

#### Section 3.4 Japanese Fonts and Typography
- When choosing fonts, it would be better to mention terms like Std and Pro for clarity (Suzuki-san). Good idea. It might be hard to include in the main text, so let’s put it in a sidebar note (Kida).
- The position of the ellipsis (should be centered) should be noted in the usage of punctuation marks. Also mention it in the fonts section.

#### Ruby
- We might want to include an overview of ruby somewhere in this chapter because ruby is one of the basic building block of Japanese text. I'll think about it (Kida).
- Ruby is increasingly being auto-generated. There are more instances where ruby is added to the author's text (Murata).

## Overall Structure and Content of jlreq-d

Proposed structure by bin-sensei
> 1. Digital Text
> 2. Characters and Fonts Used in Japanese Typesetting
> 3. Characters and Display Formats in Japanese Typesetting
> 4. Issues in Line Composition
> 5. Ruby (Furigana) Composition
> 6. Issues in Paragraph Composition
> 7. Composition of Headings, Notes, and Figures
> 8. Issues in Writing Directions (Vertical and Horizontal Writing)
> 9. Readability and Accessibility
> 10. Text Processing in Digital Texts  
> 	Appendix A: Details on Character Classes and Spacing  
> 	Appendix B: Details on Character Classes and Breakability

- Combine 2 & 3 as mentioned earlier
- Remove the term "issues" from titles.
- 7: Typesetting of Headings, Notes, and Figures: Make each a separate chapter.
- 10: Text Processing of Digital Texts: The discussion about creating text that can withstand copying and pasting or searching should be included in each chapter. If a comprehensive explanation is needed, create a separate chapter.
- Chapter numbering: Instead of the current draft format "Chapter n," use just the number "n" (bin-sensei).

## Other Topics of jlreq-d
- We want to describe not only simplified methods but also more sophisticated ones. First, describe the simplified methods, then point out their problems and explain the solution. Gradually lead to advanced methods (Kida). Encouraging readers to challenge themselves if possible (bin-sensei).
- We should decide on a method for numbering figures. In JLReq, the numbers shifted with additions and deletions, making management difficult (bin-sensei). Keep this in mind and consider it when we start adding figures (such as giving descriptive names, managing them in a searchable database, dynamically loading them, etc.) (Kida).
- It might be time to create file templates (Shmono). It's better to do this after the draft is finished. Until the user interface is resolved, we won’t mix English and Japanese texts (Kida)

## Next Meeting
June 25, two weeks later.

**Comment by kidayasuo** [comment]
# JLReq TF meeting notes 2024-6-11

## jlreq-d第2, 3章のレビュー
これらの章では、一般の書き手のためのガイドラインを示す。印刷のワークフローとは異なり、デジタル形式では書き手がテキストレイアウトの主要な部分を直接コントロールできるので、書き手のためのガイドラインは重要と考える。

#### 全般
テキストを書く人のためにカバーすることが重要だと思われることは（敏先生）：
- 約物の使い方
- 縦組横組
- 段落の問題
- 見出しと注：これは書くことが出てくれば後で
- ラテン文字、アラビア数字の使い方
全て現在の第2章の内容に入っている（木田）

2, 3章を合併する。我々の頭の中でテキストとスタイルの整理ができていればOK：小林さん

#### 3.1 段落スタイル
- ブロック段落：「段落間に空白行を入れる」→「段落間の間隔を開ける」
- ぶら下がりインデント、注と箇条書きにも使われる
- 行揃えのスタイルと段落スタイルの明確な区分けがしにくい。どう分けるべきだろうか？

#### 3.4 日本語書体とフォント
- フォントの選び方のところで、もっとわかりやすく、Std、Pro などの呼称にも言及すると良いのでは（鈴木さん）。良いアイディア。本文には入れにくいのでサイドノートに（木田）
- 三点リーダの位置は、約物の使い方の中で注意喚起。フォントのところでも。

#### ルビ
- ルビの概説をこの章のどこかに入れたいかもしれない。ちょっと考える（木田）
- どんどんルビは自動生成の方向にある。著者のテキストにないルビが増えてゆく（村田）

## jlreq-d全体の章立てと内容

敏先生の提案
> 1　デジタルテキストとは  
> 2　日本語組版に使用する文字とフォント  
> 3　日本語組版の文字と表示形式  
> 4　行の組版処理の問題点  
> 5　ルビの組版処理  
> 6　段落の組版処理の問題点  
> 7　見出し・注・図版の組版処理  
> 8　組方向（縦組と横組）の問題点  
> 9　読みやすさとアクセシビリティ  
> 10　デジタルテキストのテキスト処理  
> 附属書A　文字クラスと字間処理の詳細  
> 附属書B　文字クラスと分割の可否の詳細

- 2章と3章を統合
- 「問題点」という語句をタイトルから削る
- 7 見出し・注・図版の組版処理：それぞれ別個の章に。
- 10 デジタルテキストのテキスト処理：コピー＆ペーストや検索に耐えるテキストを作らなくてはという話。それぞれの章に内容を入れる。まとまって説明した方がよければ章を作る。
- 章の番号は、現在のドラフトの「第n章」という形式ではなく、単に数字だけの「n」が良い（敏先生）。

## jlreq-dその他の話題
- 単純化した方法だけではなく、より高度な方法も案内したい。まず単純化した方法を記述する。そしてその問題点を指摘して解決方法を示す。そうやって徐々に高度な方法に誘う、という書き方をしよう（木田）　できれば挑戦してということ（敏先生）
- 図版の番号の付け方を決めた方が良い。JLReqでは追加や削除に従ってずれて管理に困った（敏先生）。頭に置いておいて、図版を加え出したら考えよう（説明的な名前をつける、言葉から検索できるようにデータベースで管理して動的にロードする、など）（木田）
- ファイルのテンプレートをそろそろ作った方が良いのではないか（下農）ドラフトが終わってからで良いのでは。ユーザーインターフェースが解決しない限り、英語と日本語が混じった形式にはしない（木田）

## 次回のミーティング
2週間後の6月25日

**Comment by xfq** [comment]
> The position of the ellipsis (should be centered) should be noted in the usage of punctuation marks. Also mention it in the fonts section.

Should we also mention that em dashes also need to be centered? Many fonts that are not designed for CJK do not center it, and even some CJK fonts do not center it.

-----

See also [clreq](https://www.w3.org/TR/clreq/#id82).

**Reply by kidayasuo** [reply]
Yes, I agree. There are other similar characters that need to be centered, or differently aligned in the CJK context.

In the case of dashes I believe they are actually always centered or balanced against the design of the main characters of the particular font. The common issue is that often glyphs of ellipsis and dashes surface from a Latin font in the fallback list unless appropriate CJK fonts are put in the front, and required positions are different between them.

One difficulty might be that when an English text containing dashes appears in CJK context, these dashes would need to be one optimized for the font used for the English text. It means that these characters need to come up from the same font of their surrounding text. It suggests that simple font fallback list would not work.
